mr/src/test/resources/test-table/metadata/v2.metadata.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Why do we need these?

Reply:
These were initially added when we were using [HiveRunner](https://github.com/klarna/HiveRunner/) to do a proper end-to-end test with in-memory Hive and MR etc. We've now moved that code over to the `iceberg-hive` subproject another branch so I'm not sure whether the tests left here still require these files, we'll take a look at which ones can be removed.

Reply:
We generally prefer to build data files and tables in a setup method, that way we don't add binary files to this repository. This is what we do for all of the Spark tests so I'd like to use that pattern here as well.

Reply:
I completely agree. I'll try make the test look more like the one @rdsr wrote for the other InputFormat.